bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/i18n/unifi.properties

```diff
@@ -150,18 +158,9 @@ channel-type.config.unifi.poeEnable.mode.description = The value to set when set
 channel-type.config.unifi.poeEnable.mode.option.auto = Auto
 channel-type.config.unifi.poeEnable.mode.option.pasv24 = 24V
 channel-type.config.unifi.poeEnable.mode.option.passthrough = Passthrough
-channel-type.config.unifi.guestVouchersGenerate.voucherCount.label = Number
-channel-type.config.unifi.guestVouchersGenerate.voucherCount.description = Number of vouchers to create
-channel-type.config.unifi.guestVouchersGenerate.voucherExpiration.label = Expiration Time
-channel-type.config.unifi.guestVouchersGenerate.voucherExpiration.description = Minutes a voucher is valid after activation
-channel-type.config.unifi.guestVouchersGenerate.voucherUsers.label = Users
-channel-type.config.unifi.guestVouchersGenerate.voucherUsers.description = Number of users for voucher, 0 if no limit
-channel-type.config.unifi.guestVouchersGenerate.voucherUpLimit.label = Upload Speed Limit
-channel-type.config.unifi.guestVouchersGenerate.voucherUpLimit.description = Upload speed limit in kbps, no limit if not set
-channel-type.config.unifi.guestVouchersGenerate.voucherDownLimit.label = Download Speed Limit
-channel-type.config.unifi.guestVouchersGenerate.voucherDownLimit.description = Download speed limit in kbps, no limit if not set
-channel-type.config.unifi.guestVouchersGenerate.voucherDataQuota.label = Data Transfer Quota
-channel-type.config.unifi.guestVouchersGenerate.voucherDataQuota.description = Data transfer quota in MB per user, no limit if not set
+
+# channel types config
+
```

Comment:
I see you have cleaned up these unused strings, can you also remove these lines including the next one (`GENERATE`)?
```suggestion
```